validator/testcases/javascript/nodedefinitions.py

```diff
@@ -57,6 +57,9 @@ def node(branches=None, dynamic=False, action=None, returns=False,
     "FunctionExpression": node(branches=("body", ), dynamic=True,
                                action=actions._func_expr, returns=True,
                                is_block=True),
+    "ArrowExpression": node(branches=("body", ), dynamic=True,
+                            action=actions._func_expr, returns=True,
+                            is_block=True),
```

Comment:
For the purposes of the validator, it's exactly the same. There are some semantic differences, the main ones being:

* It captures the `this` object of its enclosing scope.
* It does not have access to the `arguments` object.

The main use, though, is that it's just shorter to type.